awx/settings/local_settings.py.docker_compose

```diff
@@ -196,6 +196,7 @@ LOGGING['handlers']['syslog'] = {
 LOGGING['loggers']['django.request']['handlers'] = ['console']
 LOGGING['loggers']['rest_framework.request']['handlers'] = ['console']
 LOGGING['loggers']['awx']['handlers'] = ['console', 'external_logger']
+LOGGING['loggers']['awx.main.commands.run_callback_receiver']['handlers'] = []  # propogates to awx
```

Comment:
was removed in:

https://github.com/ansible/awx/commit/da74f1d01f65cf5f0fa4c49071c866820703a3f6#diff-f5b2830c1caf5063d9a38e4b450cab43

Those facts changed with https://github.com/ansible/awx/pull/2472, and we need this again. Otherwise we'll be writing to a logfile in dev, which is not the intent.